tests/internal/anvil/framework.go

```diff
@@ -331,7 +331,7 @@ func ModifyOAuth2Provider(realm string, tokenType AccessTokenType) (original []b
 		advancedKey = "advancedOAuth2Config"
 	)
 	clientBased := tokenType != CTS
-	original, err = am.GetService(realm, oauth2Service)
+	original, _ = am.GetService(realm, oauth2Service)
```

Comment:
```suggestion
	original, err = am.GetService(realm, oauth2Service)
	if err != nil {
		return original, err
	}
```
We should probably handle this error rather than suppress it.